.gitignore

```diff
@@ -62,7 +51,7 @@ jspm_packages
 .LSOverride
 
 # Icon must end with two \r
-Icon
+Icon

 
```

Comment:
Oh! That's what the two /r means. That's obvious now, Mike and I were convinced it was a weird accident. Should I open an issue to ensure it's added back?